packages/clerk-js/src/ui/elements/TimerButton.tsx

```diff
@@ -27,7 +27,7 @@ export const TimerButton = (props: TimerButtonProps) => {
 
   const disable = () => {
     setRemainingSeconds(throttleTimeInSec);
-    intervalIdRef.current = setInterval(() => {
+    intervalIdRef.current = window.setInterval(() => {
```

Comment:
:question: I assume simply accessing `window` is safe in every context.

Reply:
Hey @gkats , thanks for checking!

It should be safe in this case, since the `disable` function is invoked within the `useEffect` callback (client-side only) or from a user interaction (clicking the button). In both cases, `window` should always exist.